src/solver.js

```diff
@@ -949,13 +956,16 @@ function Tableau(precision) {
     // Solution attributes
     this.feasible = true; // until proven guilty
     this.evaluation = 0;
+    this.simplexIters = 0;
 
     this.varIndexByRow = null;
     this.varIndexByCol = null;
 
     this.rowByVarIndex = null;
     this.colByVarIndex = null;
 
+    this.optcr = optcr || 0.1;
```

Comment:
I would make this value 0 by default (If you do not know about this option, you would want the optimal value or think the solver is broken).